distributions/openhab/src/main/resources/bin/backup

```diff
@@ -55,9 +59,16 @@ setup(){
 
   fileList="$OPENHAB_RUNTIME/bin/userdata_sysfiles.lst"
 
+  ## Parse arguments
+  if [ "$1" = "--full" ]; then
+    echo "including cache"
+    INCLUDE_CACHE="true"
+    shift
+  fi
+
   timestamp=$(date +"%y_%m_%d-%H_%M_%S")
   ## Set the filename
-  if [ -z "$1" ]; then
+  if [ -z "$1" ] || [ "$1" = "--full" ]; then
```

Comment:
With the latest changes, the addition here doesn't do anything. Probably best to keep it as:

```
if [ -z "$1" ]; then
```

Do you agree?

Reply:
Thought I changed this. How strange...